### PR TITLE
Estonian translation update

### PR DESCRIPTION
**Pull request by Madis0**
Estonian has been very inconsistent in Essentials(X) for a while. 
Since I am the primary maintainer of the language in Minecraft: Java Edition, I've decided to help a popular Bukkit/Spigot plugin out too by reviewing the consistency of the whole file, while also translating new phrases as they are added to this fork.

Some phrases' variables in the form of `{0}`, `{1}`, etc have been reordered and therefore the color tags have been matched. 
E.g. if the phrase was `§6Lorem §c{0}§6 ipsum §c{1}§6 dolor§6` and I moved the variables next to each other, it is now `§6Lorem §c{0} {1}§6 ipsum dolor§6`.

I also did my best to keep encoded and escaped letters overall work properly, but if you find an issue regarding that, I'd be glad to fix it.

### CODE REVIEW

File: Essentials/src/messages_et.properties [hunk 1]
```diff
@@ -4,616 +4,616 @@
...
 distance=\u00a76Kaugus\: {0}
 dontMoveMessage=\u00a76Teleportatsioon algab\u00a7c {0}\u00a76 p\u00e4rast. \u00c4ra liigu.
-downloadingGeoIp=GeoIP andmebaasi allalaadimine... see v\u00f5ib v\u00f5tta aega (riik\: 1.7 MB, linn\: 30MB)
-duplicatedUserdata=Topelt kasutaja andmed\: {0} ja {1}.
-durability=\u00a76Sellel esemel on \u00a7c{0}\u00a76 kasutust j\u00e4rel
-east=E
+downloadingGeoIp=Toimub GeoIP andmebaasi allalaadimine... see v\u00f5ib aega v\u00f5tta (riik\: 1.7 MB, linn\: 30MB)
+duplicatedUserdata=Topelt kasutajaandmed\: {0} ja {1}.
+durability=\u00a76Sellel esemel on \u00a7c{0}\u00a76 kasutust j\u00e4rel.
+east=ida
```
Comment (mdcfe) [inline Essentials/src/messages_et.properties:109]: Is there no way of shortening this time a single letter like in English, or does that not make sense in Estonian?

**Review by mdcfe — COMMENTED** [review]
> 428 lines changed

😮

Thanks for taking the time to submit this! Just one minor comment on the compass directions, but otherwise I can't see anything obviously wrong (not that I speak Estonian 😛).

File: Essentials/src/messages_et.properties (hunk 1 above)
Comment (Madis0) [reply Essentials/src/messages_et.properties:109]: I could attempt to do that, but it seems like a regional thing to shorten these to one letter only in English and use a full word in other languages. Besides, all words are between 3 and 5 characters :grin: 

Would it actually look bad in game though? Don't have a server to test this on at the moment.

Comment (mdcfe) [reply Essentials/src/messages_et.properties:109]: `Suund:  ida (90 kraadi).`

I suppose it looks fine in context, and it turns out the directions are only used in `/compass`.

**Review by mdcfe — DISMISSED** [review]
I'm happy to merge this if it's ready to be merged.

**Comment by Madis0** [comment]
Yes, please merge it. Would be nice if it was added to 1.13 branch too 😀

**Comment by mdcfe** [comment]
I've merged this into 1.13 builds - it should be in normal builds when the 1.13 update is officially released.

**Comment by Madis0** [comment]
Thank you! 
Unfortunately, I found a small error in the last minute, could you please replace line 494 with
```
sudoExempt=\u00a74Sa ei saa m\u00e4ngijat \u00a7c{0}\u00a74 sundida.
```
because it was missing the selector `{0}` (as seen in https://github.com/EssentialsX/Essentials/commit/48bf264178ef35711b0e90064857d17bf20aba2f)

**Comment by mdcfe** [comment]
I've fixed this in the 1.13 branch. Thanks for pointing this out.

**Pull request merged**